features/storage/blockdevice/MBRBlockDevice.h

```diff
@@ -52,21 +52,25 @@ class MBRBlockDevice : public BlockDevice {
      *
      *  @param bd       Block device to partition
      *  @param part     Partition to use, 1-4
-     *  @param type     8-bit partition type to identify partition contents
+     *  @param type     8-bit partition type or ID to identify partition contents
      *  @param start    Start block address to map to block 0 of partition.
      *                  Negative addresses are calculated from the end of the
      *                  underlying block devices. Block 0 is implicitly ignored
      *                  from the range to store the MBR.
      *  @return         0 on success or a negative error code on failure.
      *  @note This is the same as partition(bd, part, type, start, bd->size())
+     *  @note 8-bit partition type or ID is intended to specify the filesystem partition
```

Comment:
You don't need a note. Just add to the parameter description above.

Reply:
> 8-bit partition type or ID is intended to specify the filesystem partition contains and/or to flag special access methods used to access these partitions for example like CHS mapping, LBA access etc.,

I don't understand this sentence. I get confused at `contains and/or to flag`

Reply:
> 8-bit partition type or ID is intended to specify the filesystem partition contains and/or to flag special access methods used to access these partitions for example like CHS mapping, LBA access etc.,
> 
> I don't understand this sentence. I get confused at contains and/or to flag

I and Evelyne discussed the same point so decided to remove this note and only link the wiki page as like your previous comment.